### ISSUE TITLE
Flag to disable policies

### ISSUE THREAD

**Issue by lahabana** [kumahq/kuma#3278]
### Description

Currently there's no way to disable a policy without deleting them

**Comment by lahabana** [comment]
This should be considered in policy 2.0

**Comment by lahabana** [comment]
xref: #3330

**Comment by lahabana** [comment]
@lobkovilya I guess now you could do with a targetRef proxy-group pointing to nothing.

**Comment by lahabana** [comment]
@lobkovilya @michaelbeaumont wasn't the decision for this in the end was that each policy needed to be defined in a way that would enable to disable them?

**Comment by michaelbeaumont** [comment]
Potentially, though that discussion was specifically around disabling parts of a policy not overriding entire existing policy resources.

**Comment by lobkovilya** [comment]
But it's sort of the same, if you can disable parts, you can disable the whole policy. Overall this means we don't want the same `disabled` flag for all policies.

**Comment by michaelbeaumont** [comment]
Yeah I guess, though it'd be more tedious to have to disable multiple parts of a policy instead of the entire thing.

I read the title as how to disable specific, existing Policy resources (i.e. disable `timeout-1` but not `timeout-2` without deleting `timeout-1`) but seems like maybe idea here was to instead disable some configuration under `Timeout`, independent of exactly which other `Timeout` resources exist.

**Comment by lobkovilya** [comment]
Yeah, you are right, actually. These are different use cases. For example, you want this flag to disable health checks for a few minutes to check how it works without them, and you don't want to create another more specific policy to disable them. So probably both things make sense

**Comment by jakubdyszkiewicz** [comment]
> Triage: we don't need this for now
